src/AqBanking/Command/ShellCommandExecutor.php

```diff
@@ -28,4 +30,15 @@ public function execute($shellCommand)
 
         return new Result($output, $errorOutput, $returnVar);
     }
+
+
+    public function isRunning()
```

Comment:
Okay, now this is interesting. .... What are you trying to accomplish with this?
Is this actually working? The shellCommand will run in the background and will we still get the error output?